backend/src/v5/middleware/dataConverter/inputs/users.js

```diff
@@ -120,6 +120,45 @@ const validateAvatarData = async (req, res, next) => {
 	}
 };
 
+Users.validateForgotPasswordData = async (req, res, next) => {
+	const schema = Yup.object().shape({
+		user: Yup.string().required(),
+	}).strict(true).noUnknown()
+		.required();
+
+	try {
+		await schema.validate(req.body);
+
+		try {
+			const usernameOrEmail = req.body.user;
+			const { user } = await getUserByQuery({ $or: [{ user: usernameOrEmail }, { 'customData.email': usernameOrEmail }] });
+			req.body.user = user;
+		} catch {
```

Comment:
just respond `templates.ok` here and call next() after L135 instead of L140

```suggestion
                   next();
		} catch {
		   respond(req,res, templates.ok);
```